### PR TITLE
readme: add why & move architecture sections

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,57 +26,26 @@ Differentiating features include:
 [YAML]: http://yaml.org
 [Prometheus]: http://prometheus.io
 
-## Architecture
+## Why Chihaya?
 
-### Diagram
+Chihaya is built for developers who need to deploy a BitTorrent tracker, but have additional requirements than general purpose public trackers like those operated by [OpenBitTorrent].
+These requirements could be things like authentication, auditability, or features that take advantage of assumptions about their deployment environments.
+Chihaya's middleware framework enables simple and flexible integration point that abstracts the BitTorrent tracker protocols.
```
Comment (mrd0ll4r) [inline README.md]: Something is wrong with this sentence

File: README.md [hunk 2]
```diff
@@ -26,57 +26,26 @@ Differentiating features include:
...
+The most common use case for Chihaya is integration with the deployment of cloud software to private networks.
```
Comment (mrd0ll4r) [inline README.md]: That sounds weird to me. Would you describe quay as a platform for the deployment of cloud software to private networks?

File: README.md [hunk 3]
```diff
@@ -26,57 +26,26 @@ Differentiating features include:
 [YAML]: http://yaml.org
 [Prometheus]: http://prometheus.io
 
-## Architecture
+## Why Chihaya?
 
-### Diagram
+Chihaya is built for developers who need to deploy a BitTorrent tracker, but have additional requirements than general purpose public trackers like those operated by [OpenBitTorrent].
```
Comment (mrd0ll4r) [inline README.md]: ..., but have additional requirements *than* general purpose trackers?

File: README.md (hunk 2 above)
Comment (jzelinskie) [inline README.md]: yep

File: README.md (hunk 1 above)
Comment (jzelinskie) [inline README.md]: yea, it's missing the article "a"

>Chihaya's middleware framework enables **a** simple and flexible integration point that abstracts the BitTorrent tracker protocols.

rather it should probably be

>Chihaya's middleware framework offers a simple and flexible integration point that abstracts the BitTorrent tracker protocols.

File: README.md (hunk 3 above)
Comment (jzelinskie) [inline README.md]: I'm not sure what this post is suggesting.

Comment (mrd0ll4r) [inline README.md]: I'm not a native speaker, the construct "additional requirements than something" sounds strange to me, is it correct?

File: README.md (hunk 2 above)
Comment (mrd0ll4r) [inline README.md]: Huh, well then it's fine I guess :)

File: README.md (hunk 3 above)
Comment (jzelinskie) [inline README.md]: I think the fact that this sounded wrong to a non-native speaker is reason enough to reword it.

**Comment by mrd0ll4r** [comment]
I don't think you get notifications about my thumbs-up, so: I like the changes you proposed :)

**Comment by jzelinskie** [comment]
@mrd0ll4r ptal

**Comment by mrd0ll4r** [comment]
LGTM

**Pull request merged**